ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py

```diff
@@ -822,3 +822,50 @@ def test_digamma(
         on_device=on_device,
         x=x[0],
     )
+
+
+@st.composite
+def _sparsify_tensor_stg(draw):
+    dtype, tensor, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["float32", "float64", "int32", "int64"],
```

Comment:
```suggestion
            available_dtypes=helpers.get_dtypes("numeric")
```
No need to hard code the dtypes here. Ideally, this function should work with all numeric dtypes.